tests/exchangeContract.js

```diff
@@ -1,103 +1,105 @@
-import chai from 'chai';
-import moment from 'moment';
-import delay from 'delay';
-import { mochaAsync } from './utils';
-import { Application } from '..';
+import { expect, assert } from "chai";
+import moment from "moment";
+import delay from "delay";
+import { mochaAsync } from "./utils";
+import { ERC20Contract, ExchangeContract } from "..";
+var userPrivateKey =
+  "0x7f76de05082c4d578219ca35a905f8debe922f1f00b99315ebf0706afc97f132";
 
-const userPrivateKey = '0x7f76de05082c4d578219ca35a905f8debe922f1f00b99315ebf0706afc97f132';
-
-const { expect } = chai;
 const ethAmount = 0.1;
-let contractAddress = '0x949d274F63127bEd53e21Ed1Dd83dD6ACAfF7f64';
+var contractAddress = "0x949d274F63127bEd53e21Ed1Dd83dD6ACAfF7f64";
+const testConfig = {
+  test: true,
+  localtest: true, //ganache local blockchain
+};
 
-context('Exchange Contract', async () => {
+context("Exchange Contract", async () => {
   let exchangeContract;
-  let app;
   let eventId;
 
   before(async () => {
-    app = new Application({ test: true, localtest: true, mainnet: false });
+    exchangeContract = new ExchangeContract(testConfig);
+    userAddress = await exchangeContract.getUserAddress(); //local test with ganache
+    console.log("exchangeContract.userAddress: " + userAddress);
   });
 
   it(
-    'should start the Application',
+    "should start the ExchangeContract",
     mochaAsync(async () => {
-      app = new Application({ test: true, localtest: true, mainnet: false });
-      expect(app).to.not.equal(null);
-    }),
+      exchangeContract = new ExchangeContract(testConfig);
+      expect(exchangeContract).to.not.equal(null);
+    })
   );
 
   it(
-    'should deploy Exchange Contract',
+    "should deploy Exchange Contract",
     mochaAsync(async () => {
       /* Create Contract */
-      exchangeContract = app.getExchangeContract({
-        contractAddress,
-      });
+      exchangeContract = new ExchangeContract({ contractAddress });
       /* Deploy */
-      const res = await exchangeContract.deploy({});
+      let res = await exchangeContract.deploy({});
```

Comment:
this is never reassigned, so it can be a `const`

Reply:
You are right, when testing only (not in API or lib) I do not use to pay critical attention to ret vs const as we could easily reassign adding some more asserts to the testing function and a change from const to let is needed.
We can certainly use const for sure if needed no problem.